indra/tests/make_mock_ontology.py

```diff
@@ -42,7 +42,7 @@
     'UP:P15056', 'UP:O60674', 'UP:P0DP23', 'UP:Q13507', 'UP:P36507',
     'DRUGBANK:DB00305', 'HGNC:23077', 'HGNC:17347', 'EGID:27113',
     'EGID:10320', 'EGID:116986', 'EFO:0004542', 'UP:P04377',
-    'CHEBI:CHEBI:136692'
+    'CHEBI:CHEBI:136692', 'EGID:1022', 'HGNC:1778',
```

Comment:
@bgyori the test was failing becuase it wasn't able to get `HGNC:1778` when standardizing `EGID:1022`. Will this fix it? Do we need to make a new upload each time we add to this list?

Reply:
You have to run the `indra/tests/make_mock_ontology.py` script to get it up on S3 where the tests pull from.

Reply:
@bgyori okay done (though the script didn't give me any feedback about the upload being successful)! I saw this existed, but was hesitant to try it without checking. I will re-run the tests now to see if it worked.